napalm/base.py

```diff
@@ -364,3 +364,44 @@ def get_interfaces_counters(self):
             }
         """
         raise NotImplementedError
+
+        def show_route(self, destination = ''):
```

Comment:
Could you rename it to: `get_ipv4_route_to`. One to make it more human readable and two to make sure people understands they are going to get ipv4 only. We might get a `get_ipv6_route_to` some day : )